
================================================================================
global/dot-config/kitty/kitty.conf
================================================================================

diff --git a/global/dot-config/kitty/kitty.conf b/global/dot-config/kitty/kitty.conf
index 976bdfd..ba49017 100644
--- a/global/dot-config/kitty/kitty.conf
+++ b/global/dot-config/kitty/kitty.conf
@@ -6,7 +6,7 @@ enable_audio_bell       no
 repaint_delay           5
 
 # window padding       top    sides   bottom
-window_padding_width    0       5       2
+window_padding_width    0       2       0
 
 # font
 

================================================================================
global/scripts/.diff
================================================================================

diff --git a/global/scripts/.diff b/global/scripts/.diff
index ce5fa65..1981062 100755
--- a/global/scripts/.diff
+++ b/global/scripts/.diff
@@ -1,24 +1,19 @@
 #!/usr/bin/env bash
 
 cd $STOW_DIR
-if [[ $# > 0 ]]; then
-    case $1 in
-        all | a | -a | --all)
-            git diff | bat
-            ;;
-        *)
-            git diff $@ | bat
-            ;;
-    esac
-    exit 0
-fi
-
-changed=`.status -s | awk '{print $2}'`
-
-
-sel=`printf "%s\n" $changed | fzf --header="modified files" -m --bind space:toggle $FZF_INLINE`
+changed=`.status -n -s | awk '{print $2}'`
+
+case $1 in
+    all | a | -a | --all)
+        sel=`printf "%s\n" $changed`
+        ;;
+    *)
+        sel=`printf "%s\n" $changed | fzf --header="modified files" -m --bind space:toggle $FZF_INLINE`
+        ;;
+esac
 [[ -z $sel ]] && exit 1
 
+
 for f in $sel; do
     echo >> tmp.diff
     yes "=" | head -n 80 | tr "\n" "," | sed "s/,//g" >> tmp.diff

================================================================================
global/scripts/.status
================================================================================

diff --git a/global/scripts/.status b/global/scripts/.status
index dd3cd91..699a205 100755
--- a/global/scripts/.status
+++ b/global/scripts/.status
@@ -1,6 +1,16 @@
 #!/usr/bin/env bash
 
+no_fetch=false
+args=""
+for arg in $@; do
+    if [[ $arg = "--no-fetch" || $arg = "-n" ]]; then
+        no_fetch=true
+    else
+        args+="$arg "
+    fi
+done
+
 cd $STOW_DIR
-git fetch
-git status $@
+[[ $no_fetch = false ]] && git fetch
+git status $args
 

================================================================================
todo
================================================================================

diff --git a/todo b/todo
index 9281802..d959e55 100644
--- a/todo
+++ b/todo
@@ -1,5 +1,4 @@
 TODO:
-- add diff menu
 - overhaul pass managment (default get, get by name, menu)
 - overhaul polybar
     - replace search
@@ -7,5 +6,4 @@ TODO:
     - fix spotify play pause on track end
     - menus/sliders for vol and internet
     - more bars?
-- kitty/tmux bottom padding?
 - add notifications to bluetooth menu
